src/appengine/handlers/cron/fuzz_strategy_selection.py

```diff
@@ -131,7 +146,12 @@ def _query_and_upload_strategy_probabilities():
   for row in data:
     curr_strategy = data_types.FuzzStrategyProbability()
     curr_strategy.strategy_name = str(row['strategy'])
-    curr_strategy.probability = float(row['bandit_weight'])
+    curr_strategy.probability_medium_temperature = float(
+        row['bandit_weight_medium_temperature'])
+    curr_strategy.probability_high_temperature = float(
+        row['bandit_weight_medium_temperature'])
+    curr_strategy.probability_low_temperature = float(
+        row['bandit_weight_medium_temperature'])
```

Comment:
These three distributions in ndb are all initialized based on the initial temperature. This will be changed after we have one day of experimental data in our system.

Reply:
Ditto.